docs/components/early-access/alpha/bpmn-copilot/bpmn-copilot.md

```diff
@@ -0,0 +1,31 @@
+---
+id: bpmn-copilot
+title: BPMN copilot
+sidebar_label: Get started
+description: "Using AI, select users can now chat with the BPMN copilot for assistance generating new and functional BPMN process diagrams in Web Modeler based on a process description."
+---
+
+<span class="badge badge--cloud">Camunda 8 SaaS only</span>
+
+:::note Terms of use
+By using this tool, you agree to Camunda's use of the anonymized input and output data and anonymized feedback to improve it.
+:::
+
+Using AI, select users can now chat with the BPMN copilot for assistance generating new and functional BPMN process diagrams in [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md) based on a process description.
+
+## Get started
+
+:::warning
+Creating a BPMN process diagram with the BPMN copilot will overwrite existing work.
+:::
+
+To use the BPMN copilot in Web Modeler, take the following steps:
+
+1. Log in to [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md).
+2. Click **New project > Create new > BPMN diagram**. If you have opted in to test this feature, the Camunda copilot chat window will pop up.
```

Comment:
I recommend adding a link to the opt in instructions

Reply:
@christinaausley I think this would be the best link:

https://docs.camunda.io/docs/components/console/manage-organization/enable-alpha-features/#enable-ai-powered-features